src/tmle_inputs/from_actors.jl

```diff
@@ -44,7 +62,7 @@ all_variants(bqtls::DataFrame, transactors::Vector{DataFrame}) = Set(vcat(bqtls.
 
 
 read_snps_from_csv(path::Nothing) = nothing
-read_snps_from_csv(path::String) = unique(CSV.read(path, DataFrame; select=[:ID, :CHR]), :ID)
+read_snps_from_csv(path::String) = unique(CSV.read(path, DataFrame; select=[:ID, :CHR, :TF]), [:ID, :TF])
```

Comment:
So this is a breaking change, the TF column is not an option.

Reply:
How would you propose making this optional? Just read the CSV file in without selecting columns, and then check whether TF is in names(df)?

Reply:
addressed: e4c2e131323f5032ff8a3be17776638ec0b1ba05